app/controllers/passthrough_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PassthroughController < ApplicationController
+  after_action :skip_authorization
+
+  def index
+    current_organisation = document_models.find { |model| model.organisations.include? current_user.organisation_content_id }
+
+    if current_user.gds_editor?
+      redirect_to "/aaib-reports"
+    elsif current_organisation
+      redirect_to "/" + current_organisation.slug
+    else
+      redirect_to error_path
```

Comment:
Rather than redirecting to an empty error action why not give an appropriate status (possibly a 403) and message?
eg. 
```ruby
status_code = 403
render status: status_code, text: "#{status_code} error"
```